srcpkgs/tuner/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'tuner'
+pkgname=tuner
+version=1.5.1
+revision=1
+build_style=meson
+hostmakedepends="glib gettext pkg-config vala"
+makedepends="glib geoclue2-devel geocode-glib-libsoup2-devel glib-devel
```

Comment:
I am against including a new package that uses libsoup2, especially geocode-glib-libsoup2, we should be trying to slowly get rid of those packages.

Reply:
Things should be resolved in next versions, as the dev seems to be on board about dropping support for geocode-glib al together in the new milestone versions 1.6.

https://github.com/louis77/tuner/issues/106

So, this would only be a temporary solution, just to get Tuner in the src collection.

Sorry for the delay, I don't have a lot of free time currently, I'll try and get this done next week or so.